components/list/list-controls.js

```diff
@@ -19,31 +20,47 @@ export class ListControls extends SelectionControls {
 				--d2l-selection-controls-padding: var(--d2l-list-controls-padding, 18px);
 				z-index: 6; /* must be greater than d2l-list-item-active-border */
 			}
+			:host([child-has-color]) {
+				margin: 0 -0.9rem;
+			}
 			:host([no-sticky]) {
 				z-index: auto;
 			}
-			.d2l-list-controls-extend-separator {
-				padding: 0 0.9rem;
+			.d2l-list-controls-color,
+			.d2l-list-controls-extend-separator  {
+				padding: 6px 1.8rem;
```

Comment:
I found updating the spacing to this caused less issues with the hover state appearing beyond the sticky header